weldx/tests/test_measurement.py

```diff
@@ -454,7 +455,7 @@ def test_get_signal_data(self):
         mc.add_transformation(self._default_transformation(), data=data)
         mc.create_transformation("transformation_2", None, output_signal_unit="A")
 
-        assert mc.get_signal_data("transformation").identical(data)
```

Comment:
I am not really sure why this worked in the first place since `get_signal_data` is supposed to return a `TimeSeries` (which doesn't have `.identical`) 🤔

Reply:
Is that what the type hint says or the actual code. Maybe the type hints or the code need an adjustment too.

Reply:
the type hint says `TimeSeries` but I didn't check any further (the code looks more like `xarray`)

Reply:
Hmmm... strange. Will have to check this on my own. Usually, this should be caught by `mypy`